tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/Main.java

```diff
@@ -0,0 +1,161 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import com.google.common.base.Optional;
+import fi.helsinki.cs.tmc.langs.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.RunResult;
+import fi.helsinki.cs.tmc.stylerunner.validation.ValidationResult;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Main {
+
+    final static String EXERCISE_PATH = "exercisePath";
+    final static String OUTPUT_PATH = "outputPath";
+    private static TaskExecutor executor = new TaskExecutorImpl();
+
+    public Main(TaskExecutor executor) {
+        Main.executor = executor;
+    }
+
+    public static void main(String[] args) {
+        if (args == null || args.length == 0) {
```

Comment:
In normal cases it can't, but if it's called from somewhere else than command line it can be. We can remove or leave the null check.